src/widgets/restFullClient/model/requestHandler.ts

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios';
+import { IRequestData } from './requestHandlerModel';
+
+export const sendRequest = async (DATA: IRequestData) => {
+  const { method, url, headers, body } = DATA;
```

Comment:
```suggestion
export const sendRequest = async ({ method, url, headers, body }: IRequestData) => {
```

Кроме того можнно поправить пропсы:
```suggestion
export const sendRequest = async ({ method, url, headers, data}: IRequestData) => {
```

Reply:
исправил